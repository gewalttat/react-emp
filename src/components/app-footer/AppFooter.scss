.footer {
    height: 70px;
    display: flex;
    align-items: center;
    justify-content: center;
    background: #424242;
}

.footer-text {
    width: 151px;
    height: 24px;
    font-family: Montserrat, sans-serif;
    font-style: normal;
    font-weight: 300;
    font-size: 20px;
    line-height: 24px;
    color: #f65261;
  
    &__bold {
      font-weight: 700;
    }
  }