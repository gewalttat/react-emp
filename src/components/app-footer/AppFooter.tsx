import React, { FC } from 'react';
import './AppFooter.scss'

export const AppFooter: FC = () => {
  return (
    <div className='footer'>
      <div className='footer-text'>
        <span className='footer-text__bold'>netflix</span> <span>roulette</span>
      </div>
    </div>
  )
}