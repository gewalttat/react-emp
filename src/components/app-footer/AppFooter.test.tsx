import React from 'react';
import renderer from 'react-test-renderer';

import {AppFooter} from './AppFooter';

it('renders correctly', () => {
  const tree = renderer.create(<AppFooter/>).toJSON();
  expect(tree).toMatchSnapshot();
});