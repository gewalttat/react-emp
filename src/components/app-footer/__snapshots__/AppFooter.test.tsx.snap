// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="footer"
>
  <div
    className="footer-text"
  >
    <span
      className="footer-text__bold"
    >
      netflix
    </span>
     
    <span>
      roulette
    </span>
  </div>
</div>
`;
